orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -474,6 +475,41 @@ void addObjectMethodsForTraining(py::module& m) {
         return static_cast<PipelineTrainingSession*>(sess->GetSessionHandle())->IsGraphOutputFp32Node(output_name);
       });
 
+
+struct PyTrainingAgent {
+  PyTrainingAgent(PyInferenceSession* session) {
+    agent_ = onnxruntime::make_unique<TrainingAgent>(session->GetSessionHandle());
+  }
+
+  TrainingAgent* GetAgentHandle() const { return agent_.get(); }
+  virtual ~PyTrainingAgent() {}
+
+ private:
+  std::unique_ptr<TrainingAgent> agent_;
+};
+
+py::class_<PyTrainingAgent>(m, "TrainingAgent", R"pbdoc(This is the main class used to run a ORTModule model.)pbdoc")
```

Comment:
I am naïve on pybind here... I am just curious if it possible we directly bind with TrainingAgent class? 
IIUC, PyInferenceSession is needed to accommodate custom op library management for the Python user. 

I am NOT say we should bind with TrainingAgent, I like how PyTrainingAgent is matching the layering structure as PyInferenceSession.

Reply:
Yes, we could and I am ok in doing if would like better.

PyTrainingAgent is a leftover from my previous approach in which I encapsulated both TrainingAgent and PyInferenceSession inside PyTrainingAgent.

Reply:
removing it